app/assets/javascripts/collections/user.js

```diff
@@ -0,0 +1,14 @@
+
+(function(App) {
+
+  'use strict';
+
+  App.Collection.User = App.Collection.Table.extend({
+
+    // url: 'admin/users.json'
+
+    url: '/users.json'
```

Comment:
My implementation? I only have to replace one url with another, what implementation? It's exactly the same we are using, just changing the endpoint.